iatBatch.go

```diff
@@ -45,6 +46,23 @@ func NewIATBatch(bh *IATBatchHeader) IATBatch {
 	return iatBatch
 }
 
+// UnmarshalJSON un-marshals JSOn IATBatch
+func (batch *IATBatch) UnmarshalJSON(p []byte) error {
```

Comment:
Can we add a `if batch == nil` check and set the value? `*batch = NewIATBatch()`

Otherwise this method would fail on `var batch *IATBatch` I believe.

Reply:
I will check.  I didn’t think it compiled when I had the nil check.    I’ll review.